webpack-watcher.js

```diff
@@ -0,0 +1,57 @@
+const childProcess = require('child_process');
+
+module.exports = (options) => {
+   console.log("Forking " + options.username);
+   const child = childProcess.fork(__dirname + '/webpack-compiler.js')
+
+   let bundledCallback = () => {}
+   let watchers = [];
```

Comment:
This could be a `const` if we emptied the list of watchers instead of replacing it. I think that would be a little more true to the way it's used. Unfortunately, there isn't a super intuitive way to empty an array (unless you think `watchers.length = 0` is intuitive), but if we replaced this with a `Set`, then we could do `watchers.clear()`.